conjure-lib/src/main/java/com/palantir/conjure/java/lib/internal/ConjureCollections.java

```diff
@@ -126,4 +135,43 @@ public static <T> Set<T> newNonNullSet(Iterable<? extends T> iterable) {
 
         return set;
     }
+    // This method returns a list that can't handle nulls. Do not use this unless the nonNullCollections flag is set
```

Comment:
odd missing whitespace above this, I'm not sure why the formatter didn't insert a newline